go/src/koding/klient/machine/transport/rsync/rsync.go

```diff
@@ -131,6 +139,11 @@ func (c *Command) run(ctx context.Context, scan func(r io.Reader)) error {
 
 	// Add default arguments.
 	c.Cmd.Args = append(c.Cmd.Args, "-zlptgoDd")
+	if c.Cmd.Env == nil {
+		c.Cmd.Env = englishEnv
+	} else {
+		c.Cmd.Env = append(englishEnv, c.Cmd.Env...)
```

Comment:
```
WARNING: DATA RACE
Write at 0x00c420224a10 by goroutine 11:
  runtime.slicecopy()
      /usr/local/go/src/runtime/slice.go:160 +0x0
  koding/klient/machine/transport/rsync.(*Command).run()
      /root/koding/go/src/koding/klient/machine/transport/rsync/rsync.go:145 +0x431
```

Btw you may want to use `(koding/os.Environ).Encode()`, in case englishEnv has already LC_ALL defined c.Cmd.Env won't overwrite it.